### DISCUSSION TITLE
Issues with start time for Timelapse – internal camera time change

### DISCUSSION THREAD

**Discussion by Kim-Ocean** [gopro/labs#385] [Q&A]
I would like to run my Hero 11 on Timelapse Photo mode for 9 h from 9 am to 6pm (9:00 – 18:00) in a interval of 2 sec. In addition, I would like to repeat that for several days. I am using the Hero 11 without battery connected via a USB Battery Pack Keep-Alive load (Sotobeams) to a powerbank.
I am trying to use the following QR code created with the QR Control App:

!09:00NmTPp.2fL!S!32400E!R

Unfortunately when the GoPro reads the code somehow the internal clock of the camera changes to 9:26 no matter what the actual time is. I am using the camera with the 24 hour clock. I also tried to set on GPS with the QR code but the same mistake kept happening.
Does anybody know what I am doing wrong or how I can solve the issue that the camera keeps the right time?

Thank you very much for your help!

**Comment by dnewman-gpsw** [comment]
I think the camera is not keeping RTC (Real-time clock) refreshed from USB power. I will have to ask someone in the hardware team for clarification as I hear of this more often. Camera with battery is super reliable, but it you sleep it too long without a battery, this error can occur.
